app/authentication.js

```diff
@@ -1,32 +1,59 @@
 // Configure login mechanism
+const session = require('express-session')
 const passport = require('passport')
 const config = require('../config')
+const SamlStrategy = require('passport-saml').Strategy
 
-var SamlStrategy = require('passport-saml').Strategy
+const getUserRoleByUsername = require('./services/data/get-user-role-by-username')
 
 module.exports = function (app) {
   if (!config.AUTHENTICATION_ENABLED) return
 
   passport.serializeUser(function (user, done) {
```

Comment:
If the URL isn't present none of the Passport methods are called. The Browser throws an error message stating it can't reach the url. Its tricky to log into error within passport.